docs/REST API/rest-api.md

```diff
@@ -2036,6 +2037,18 @@ If `after` is being used with `created_at` or `updated_at`, the table must be sp
           ],
           "installed_paths": ["/usr/lib/some-path-1"]
         }
+      ],
+      "policies": [
+        {
+          "id": 1,
+          "name": "Gatekeeper enabled",
+          "query": "SELECT 1 FROM gatekeeper WHERE assessments_enabled = 1;",
+          "description": "Checks if gatekeeper is enabled on macOS devices",
+          "resolution": "Fix with these steps...",
+          "platform": "darwin",
+          "response": "fail",
+          "critical": false
+        }
```

Comment:
Since full policy details may not be necessary for users of this endpoint, could potentially simplify to:
```suggestion
        {
          "id": 1,
          "name": "Gatekeeper enabled",
          "response": "fail",
          "critical": false
        }
```